src/joystick/windows/SDL_rawinputjoystick.c

```diff
@@ -841,6 +842,17 @@ static SDL_RAWINPUT_Device *RAWINPUT_DeviceFromHandle(HANDLE hDevice)
     return NULL;
 }
 
+static int GetSteamVirtualGamepadSlot(Uint16 vendor_id, Uint16 product_id, const char *device_path)
+{
+    int slot = -1;
+
+    if (vendor_id == USB_VENDOR_VALVE &&
+        product_id == USB_PRODUCT_STEAM_VIRTUAL_GAMEPAD) {
+        (void)SDL_sscanf(device_path, "\\\\.\\pipe\\HID#VID_045E&PID_028E&IG_00#%*X&%*X&%*llX#%d#%*u", &slot);
```

Comment:
maybe add comment link to https://partner.steamgames.com/doc/features/steam_controller/steam_input_gamepad_emulation_bestpractices here?